connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -360,7 +360,7 @@ message StreamingQueryManagerCommandResult {
     // (Required) The id and runId of this query.
     StreamingQueryInstanceId id = 1;
     // (Optional) The name of this query.
-    string name = 2;
+    optional string name = 2;
```

Comment:
`optional` is needed to use the `hasName` method in scala

Reply:
We can also check if the value empty and treat it as missing. Not a fan of heavy use of optional keyword (Proto3 reluctantly supports it).

Reply:
I see. But then if a user deliberately set the query name to be `""`, it would be treated as null also. We are fine with that?

Reply:
I think so, yes.